### PR TITLE
edit Troubleshooting for ArchLinux

### PR DESCRIPTION
**Pull request by laptander**
Add some suggestions and variable to choose audio codec

### CODE REVIEW

File: prepare-videochat.sh [hunk 1]
```diff
@@ -130,6 +137,9 @@ HEIGHT=480
 # Frame rate of video
 GST_FPS=5
 
+# Choose audio codec from wav, aac or opus
+AUDIO_CODEC=opus
```
Comment (bluezio) [inline prepare-videochat.sh]: Could this go in a different commit?

Comment (laptander) [inline prepare-videochat.sh]: You want me to change default codec back to wav or just remove this variable?

Comment (bluezio) [inline prepare-videochat.sh]: No, opus is fine - just make the changes to AUDIO_CODEC in a different
commit, to make the change more explicit to someone who is skimming through
the history :-). Thanks!

Comment (laptander) [inline prepare-videochat.sh]: Ok, I will redo commits. Closing this PR.
